### PR TITLE
Translated into Russian

### PR DESCRIPTION
**Pull request by Miros77**

### CODE REVIEW

File: src/main/resources/assets/mechanicaltech/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "block.mechanicaltech.turbine": "Турбина",
+  "block.mechanicaltech.waterwheel": "Водяное Колесо",
+  "block.mechanicaltech.windsail": "Парусник",
```
Comment (EngiN33R) [inline src/main/resources/assets/mechanicaltech/lang/ru_ru.json]: This should probably be "Парус" or "Паруса", seeing as "парусник" is a sailing ship.

File: src/main/resources/assets/mechanicaltech/lang/ru_ru.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "block.mechanicaltech.manual_crank": "Ручной Кривошип",
+
+  "block.mechanicaltech.battery_base": "Батарейная Основа",
+  "block.mechanicaltech.battery_mid": "Батарейное Ядро",
+  "block.mechanicaltech.battery_cap": "Крышка Батареи",
```
Comment (EngiN33R) [inline src/main/resources/assets/mechanicaltech/lang/ru_ru.json]: Ideally, all three should have a single format - either adjective + noun, or noun + genitive noun. I think Ядро батареи and Основа батареи sound better than the current translation.

File: src/main/resources/assets/mechanicaltech/lang/ru_ru.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  "item.mechanicaltech.manual_crank": "Ручной Кривошип",
+
+  "item.mechanicaltech.mill": "Мельница",
+  "item.mechanicaltech.grinder": "Дробитель",
+  "item.mechanicaltech.sawmill": "Лесопилка",
+
+  "item.mechanicaltech.angled_connector_down": "Угловой Соединитель (Вниз)",
+  "item.mechanicaltech.angled_connector_right": "Угловой Соединитель (Сбоку)",
+  "item.mechanicaltech.angled_connector_up": "Угловой Соединитель (Вверх)",
+  "item.mechanicaltech.cross_connector": "Cross Connector",
```
Comment (EngiN33R) [inline src/main/resources/assets/mechanicaltech/lang/ru_ru.json]: Missed one.

File: src/main/resources/assets/mechanicaltech/lang/ru_ru.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  "block.mechanicaltech.battery_base": "Батарейная Основа",
+  "block.mechanicaltech.battery_mid": "Батарейное Ядро",
+  "block.mechanicaltech.battery_cap": "Крышка Батареи",
+
+  "block.mechanicaltech.mill": "Мельница",
+  "block.mechanicaltech.grinder": "Дробитель",
+  "block.mechanicaltech.sawmill": "Лесопилка",
+
+  "block.mechanicaltech.angled_connector_down": "Угловой Соединитель (Вниз)",
+  "block.mechanicaltech.angled_connector_right": "Угловой Соединитель (Сбоку)",
```
Comment (EngiN33R) [inline src/main/resources/assets/mechanicaltech/lang/ru_ru.json]: Maybe this should be "Вбок", to have them all look the same. Alternatively, they could be called нижний, верхний and боковой.

**Review by EngiN33R — CHANGES_REQUESTED** [review]
Vanilla Minecraft uses regular Russian sentence case for item names. I think this translation should follow suit.
image

**Comment by Miros77** [comment]
Thanks, I'll try to fix it

File: src/main/resources/assets/mechanicaltech/lang/ru_ru.json (hunk 4 above)
Comment (Miros77) [reply src/main/resources/assets/mechanicaltech/lang/ru_ru.json]: Yes I made a mistake here

**Comment by Miros77** [comment]
Bonus

New Model for Windsail

Screenshots


Model
[windsail.zip](https://github.com/Eng1N33R/MechanicalTech/files/5669077/windsail.zip)

**Review by EngiN33R — APPROVED** [review]

**Comment by EngiN33R** [comment]
Thank you for your contribution! I've replaced the model, but in the future, it is best to use separate pull requests for separate contributions to keep things in order :)

**Pull request merged**